Zilon.Core/Zilon.Core/Tactics/Sector.cs

```diff
@@ -265,10 +266,23 @@ private void ActorManager_Added(object sender, ManagerItemsChangedEventArgs<IAct
             }
         }
 
+        private void ActorManager_Remove(object sender, ManagerItemsChangedEventArgs<IActor> e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'sender'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)